relay-metrics/src/aggregation.rs

```diff
@@ -3084,8 +3096,10 @@ mod tests {
         insta::assert_debug_snapshot!(output, @r###"
         [
             "metrics.buckets.per_batch:1|h",
+            "metrics.buckets.partition_keys:59|h",
             "metrics.buckets.batches_per_partition:1|h",
             "metrics.buckets.per_batch:1|h",
+            "metrics.buckets.partition_keys:62|h",
```

Comment:
Does order of this keys matter?

Reply:
I forgot to update this test, the `partition_keys` metric is now logged before the other ones.

But good point: the order in which the partition keys appear depends on the random seed of the `Aggregator::buckets` hash map, so the test has to account for that. I added a comment to the test case.